### PR TITLE
API-29630 add wsEndpoint option

### PR DESCRIPTION
**Pull request by johnmweisz**
- adds wsEndpoint option so that docker can connect to chrome debug on Macs

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,7 @@
-FROM vasdvp/lighthouse-node-application-base:node16 as chromebase
+FROM vasdvp/lighthouse-node-application-base:node16
 
 USER root
 
-# Install chrome dependencies
-RUN dnf -y install https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
-
-FROM chromebase as appbase
-
```
[the remark this answers is not in the document]
Comment (crolarlibertyva) [reply Dockerfile:9]: Confirmed in my amd64 testing.
```
Error: Failed to launch the browser process!
/home/node/.cache/puppeteer/chrome/linux-1095492/chrome-linux/chrome: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory


TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md

    at onClose (/home/node/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserRunner.js:300:20)
    at Interface.<anonymous> (/home/node/node_modules/puppeteer-core/lib/cjs/puppeteer/node/BrowserRunner.js:288:24)
    at Interface.emit (node:events:525:35)
    at Interface.close (node:readline:590:8)
    at Socket.onend (node:readline:280:10)
    at Socket.emit (node:events:525:35)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
    ```

**Review by crolarlibertyva — CHANGES_REQUESTED** [review]
Chrome dependencies are still needed for vanilla non-mac docker runs.

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (johnmweisz) [reply Dockerfile:9]: this download doesn't work for me, I needed to remove it to get docker to build.

[the remark this answers is not in the document]
Comment (johnmweisz) [reply Dockerfile:9]: I will add it back but I don't think this is an optimal solution

[the remark this answers is not in the document]
Comment (johnmweisz) [reply Dockerfile:9]: this suggests an alternative, how can I reproduce this error? https://stackoverflow.com/questions/66214552/tmp-chromium-error-while-loading-shared-libraries-libnss3-so-cannot-open-sha

[the remark this answers is not in the document]
Comment (crolarlibertyva) [reply Dockerfile:9]: Yeah, give that a try. We only need the dependencies. Might as well be direct about it.

[the remark this answers is not in the document]
Comment (johnmweisz) [reply Dockerfile]: reverting this and marking resolved, will review dockerfile changes at another time.

**Review by crolarlibertyva — APPROVED** [review]

**Pull request merged**